### PR TITLE
[IMP] localizations: adding improvements to mexico accounting doc task#24058

### PR DESCRIPTION
**Pull request by goliveirab**
The following improvements are done:
1. Must set the Tag field in Taxes.
2. Explanation and solution of the error:`The cfdi generated is not valid`.
3. Update finkok information link.

Fix: https://github.com/Vauxoo/enterprise/issues/283

### CODE REVIEW

**Comment by goliveirab** [comment]
@luistorresm 
Could you check this? 
Regards.

File: accounting/localizations/mexico.rst [hunk 1]
```diff
@@ -118,6 +118,11 @@ the entry with this name does not exist).
 
 Important considerations when yo enable the CFDI 3.3
 ====================================================
+Your taxes must have a "Tag" field set. If the tax is a sale tax the "Tag" field
```
Comment (luistorresm) [inline accounting/localizations/mexico.rst]: I think that this must be a new point in the configuration.

> 7. Configure the tag in the sale tags, the tax is used to....
> 
>      Note the default taxes already are configured, but in the new taxes must set the tag

File: accounting/localizations/mexico.rst [hunk 2]
```diff
@@ -586,9 +591,31 @@ FAQ
...
+
+.. tip::
+   **Solution:** This error can be shown when a database backup is done and it is
+   restore in other server, or when the XSD files are not correctly downloaded.
+
+   - Go to :menuselection:`Settings --> Technical --> Actions --> Server Actions`
+   - Look for the Action called "Download XSD files to CFDI"
+   - Click on button "Create Contextual Action"
+   - Go to the company form :menuselection:`Settings --> Users&Companies --> Companies`
+   - Open any company you have.
```
Comment (luistorresm) [inline accounting/localizations/mexico.rst]: Go to the company where you have the error.

**Comment by goliveirab** [comment]
@luistorresm 
could you check this changes?
Thanks

**Review by luistorresm — APPROVED** [review]
@nhomar 

What do you think about this PR?

Regards

**Comment by goliveirab** [comment]
@luistorresm 
Could you check this?
The reference about error: `The cfdi generated is not valid`, was moved as a note in **Enabling Explicit errors on the CFDI using the XSD local validator (CFDI 3.3)**

**Comment by luistorresm** [comment]
Odoo PR https://github.com/odoo/documentation-user/pull/265

**Comment by luistorresm** [comment]
Merged in Odoo

**Pull request closed without merging**